pycqed/instrument_drivers/physical_instruments/ZurichInstruments/UHFQuantumController.py

```diff
@@ -734,14 +920,48 @@ def acquisition_poll(self, samples, arm=True,
 
         return data
 
+    def acquisition_get(self, samples, arm=True,
```

Comment:
This seems to be an additional function for retrieving data from the UHFQA (the other being acquisition_poll). It is not entirely clear to me in how they differ and in which instances one or the other routine should be called (the default one still being acquisition_poll, see self.acquisition()).

Note: Currently we implement the polling in the UHFQC_Base class. As it accesses private attributes, we might consider using the routines provided in the ZI driver.